switch/zziplib/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# Maintainer: COMP Omega <compomega@tutanota.com>
+pkgname=switch-zziplib
+pkgver=0.13.69
+pkgrel=1
+pkgdesc='The ZZIPlib provides read access on ZIP-archives and unpacked data. It features an additional simplified API following the standard Posix API for file access'
+arch=('any')
+url='https://github.com/gdraheim/zziplib/'
+license=('LGPL')
+options=(!strip libtool staticlibs)
+source=("https://github.com/gdraheim/zziplib/archive/v${pkgver}.tar.gz")
+makedepends=('devkitA64' 'devkitpro-pkgbuild-helpers' 'switch-zlib')
+sha256sums=('846246d7cdeee405d8d21e2922c6e97f55f24ecbe3b6dcf5778073a88f120544')
+groups=('switch-portlibs')
+
+build() {
+  cd zziplib-$pkgver
+
+  source /opt/devkitpro/devkita64.sh
+  source /opt/devkitpro/switchvars.sh
+
+  ./configure --prefix="${PORTLIBS_PREFIX}" --host=aarch64-none-elf \
+    --disable-shared --enable-static --with-zlib="${PORTLIBS_PREFIX}"
```

Comment:
is the --with-zlib option necessary?

In theory it should just pick up zlib from the flags we set via switchvars.sh.